onebusaway-android/build.gradle

```diff
@@ -27,11 +27,14 @@ apply plugin: 'com.android.application'
 
 repositories {
     jcenter()
-    mavenLocal()
     maven {
         // OBA Releases - for comparator to sort alphanumeric routes
         url "http://nexus.onebusaway.org/nexus/content/repositories/releases"
```

Comment:
Got it. Hopefully Amazon can add it to JCenter or Maven Central since binaries in git isn't the best idea ever and so we wouldn't have to deal with doing there updates.